Whole-game.Rmd

```diff
@@ -109,300 +125,375 @@ create <- !is.na(where)
 
 where <- switch(
   where,
-  tmp_user = fs::path_home("tmp"),
-  tmp_session = fs::path_temp(),
+  tmp_user = path_home("tmp"),
+  tmp_session = path_temp(),
   NULL
 )
 
-foopath <- path(where, "foofactors")
+pkgpath <- path(where, "regexcite")
 
 if (!is.null(where)) {
-  if (requireNamespace("foofactors", quietly = TRUE)) {
-    remove.packages("foofactors")
-    unloadNamespace("foofactors")
+  if (requireNamespace("regexcite", quietly = TRUE)) {
+    remove.packages("regexcite")
+    unloadNamespace("regexcite")
   }
-  if (fs::dir_exists(foopath)) {
-    fs::dir_delete(foopath)
+  if (dir_exists(pkgpath)) {
+    dir_delete(pkgpath)
   }
-  fs::dir_create(where)
+  dir_create(where)
 }
 ```
 
-:::downlit
-```{r create-package, eval = create, echo = debug, R.options = list(usethis.description = NULL)}
-create_package(foopath, open = FALSE, rstudio = TRUE)
-proj_set(foopath)
+```{r create-package, eval = create, echo = debug}
+withr::with_options(
+  list(usethis.description = NULL),
+  create_package(pkgpath, open = FALSE, rstudio = TRUE)
```

Comment:
usethis's messaging is not rendered in color, so I've clearly turned too many things off or they need to be revised for the current downlit world.

Reply:
Compare how this chunk looks in the preview of this PR to how it currently looks in the main site:
https://r-pkgs.org/whole-game.html#create_package

Reply:
I've put all the `:::downlit` divs back. It seems they are still necessary for a chunk with `echo = FALSE`.

Reply:
Oh yeah, that must be why they edited them in the first place.